src/server/routes/revision.js

```diff
@@ -237,14 +238,25 @@ router.get('/:id', (req, res, next) => {
 					formatWorkChange
 				)
 			);
-			const props = {revision: revision.toJSON(), diffs};
-			res.render('page', {
+			const props = Object.assign({}, req.app.locals, res.locals, {
+				revision: revision.toJSON(),
 				title: 'RevisionPage',
-				markup: ReactDOMServer.renderToString(RevisionPage(props))
+				diffs
 			});
+			const markup = ReactDOMServer.renderToString(
+				<Layout {...props}>
+					<RevisionPage />
```

Comment:
Yeah this is fine since the Layout props will propagate down to the RevisionPage due to the way the Layout component handles its children. I agree we should come back to that though (as mentioned in IRC) - if we can give those props to the child directly we should do.